src/ws/service/utils.rs

```diff
@@ -0,0 +1,9 @@
+/// Builds a room alias name from a vector of user addresses by sorting them and joining them with a "+" separator.
```

Comment:
nit: Could this method be in `synapse_handler`? I'm not a big fan of the `utils.rs` files